spec/services/encryption/multi_region_kms_client_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+describe Encryption::MultiRegionKMSClient do
+  before do
+    stub_mapped_aws_kms_client(
+      'a' * 3000 => 'kms1',
+    )
+
+    allow(FeatureManagement).to receive(:use_kms?).and_return(kms_enabled)
+  end
+
+  let(:plaintext) { 'a' * 3000 }
+  let(:encryption_context) { { 'context' => 'attribute-bundle', 'user_id' => '123-abc-456-def' } }
+
+  kms_regions = JSON.parse(Figaro.env.aws_kms_regions)
+
+  let(:regionalized_kms_ciphertext) do
+    region_hash = {}
+    kms_regions.each do |r|
+      region_hash[r] = 'kms1'
+    end
+    { regions: region_hash }.to_json
+  end
+
+  let(:legacy_kms_ciphertext) do
+    'kms1'
+  end
+
+  let(:kms_enabled) { true }
+
+  aws_key_id = Figaro.env.aws_kms_key_id
```

Comment:
ditto --- this should be a `let` and I think it might be worth replacing with a fake value

Reply:
+1 to making sticking this inside a `let`